src/verification_key.rs

```diff
@@ -68,21 +69,11 @@ impl From<VerificationKeyBytes> for [u8; 32] {
 ///
 /// ## Zcash-specific consensus properties
 ///
-/// Ed25519 checks are described in [§5.4.5][ps] of the Zcash protocol
-/// specification. However, it is not clear that the protocol specification
-/// matches the implementation in `libsodium` `1.0.15` used by `zcashd`. Note
-/// that the precise version is important because `libsodium` changed validation
-/// rules in point releases.
+/// Ed25519 checks are described in [§5.4.5][ps] of the Zcash protocol specification and in
+/// [ZIP 215].  The verification criteria for an (encoded) verification key `A_bytes` are:
 ///
-/// The spec says that a verification key `A` is
-///
-/// > a point of order `l` on the Ed25519 curve, in the encoding specified by…
-///
-/// but `libsodium` `1.0.15` does not check this. Instead it only checks whether
-/// the encoding of `A` is an encoding of a point on the curve and that the
-/// encoding is not all zeros. This implementation matches the `libsodium`
-/// behavior.  This has implications for signature verification behaviour, as noted
-/// in the [`VerificationKey::verify`] documentation.
+/// * `A_bytes` MUST be an encoding of a point `A` on the twisted Edwards form of
+///   Curve25519, and non-canonical encodings MUST be accepted;
 ///
 /// [ps]: https://zips.z.cash/protocol/protocol.pdf#concretejssig
```

Comment:
This moved to https://zips.z.cash/protocol/protocol.pdf#concreteed25519

Reply:
The old link is still valid (I try to keep links valid unless there's been a really big refactoring), but yeah.

Reply:
I've updated the links anyways to point to the new version.